src/skmatter/clustering/_quick_shift.py

```diff
@@ -17,8 +19,8 @@ class QuickShift(BaseEstimator):
 
     Parameters
     ----------
-    dist_cutoff2 : float, default=None
-        The squared distance cutoff for searching for the next point. If None, the
+    dist_cutoff_sq : float, default=None
+        The squared distance cutoff for searching for the next point. If `None`, the
```

Comment:
Single ` in rst are italic. If you want them to rendered as code use a double ``. On top for default Python objects like `None`, `True` we can use intersphinx to link to the Python docs.

```suggestion
        The squared distance cutoff for searching for the next point. If :obj:`None`, the
```